packages/website/src/components/Faq.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { Flex } from '@coderscamp/ui/components/Flex';
+import { QuestionsAccordion } from '@coderscamp/ui/components/Question';
+import { Typography } from '@coderscamp/ui/components/Typography';
+
+const ITEMS = [
```

Comment:
🙃 In the case of non-primitive values (and functions), there is no need to write them UPPERCASE. The same applies to the `ARGS` constant in Question stories.